lib/node_modules/@stdlib/blas/ext/base/dcusumkbn/manifest.json

```diff
@@ -40,7 +40,6 @@
       "dependencies": [
         "@stdlib/napi/argv",
         "@stdlib/napi/argv-int64",
-        "@stdlib/napi/argv_double",
```

Comment:
This was the wrong package name. Notice the snakecase convention of the other packages in this list.

Reply:
Ok, got it. Thanks for the clarification.